### ISSUE TITLE
Release 1.0

### ISSUE THREAD

**Issue by ekes** [localgovdrupal/localgov_examples#6]
I've updated this to be D10 compatible. We should make a 1.0 release. I've pushed a release-1.x branch.

Can make another issue then to discuss adding it to `require-dev` for the project.

Then can add some of the other example modules (see issues).

**Comment by ekes** [comment]
Putting this here for feedback / checking.

**Comment by andybroomfield** [comment]
yep, lets make this a 1.x release and put in require-dev of the project or profile.
